.travis.yml

```diff
@@ -133,6 +133,9 @@ before_script:
           srcdir=.;
           top=.;
       fi
+    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+          CONFIG_OPTS="$CONFIG_OPTS -Wno-format";
+      fi
```

Comment:
*Formally* speaking one can't preclude possibility that there is minw compiler for MacOS X. I mean there actually is one out there. But we have intention to use it on Travis. So on practical level it works out, but *otherwise* it's not exactly right spot for the check :-)

Reply:
Just in case it wasn't obvious it was a *side* note!